g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/TranslationProductComponentKeyAction.java

```diff
@@ -175,5 +179,17 @@ public APIResponseDTO getMultTransByGet(String productName, String version, Stri
        	return super.handleResponse(APIResponseStatus.OK, compDTo);
        }
 	}
+  
+	protected void validateSourceSet(List<KeySourceCommentDTO> sourceSet) throws ValidationException {
+	
+		for(KeySourceCommentDTO ksc:sourceSet) {
+			String sft = ksc.getSourceFormat();
+			if (!StringUtils.isEmpty(sft) && !ConstantsSupportList.SOURCE_FORMAT_LIST.contains(sft)) {
+			   throw new ValidationException(ValidationMsg.SOURCEFORMAT_NOT_VALIDE);
```

Comment:
the message of SOURCEFORMAT_NOT_VALIDE should be updated to tell the API caller the enumeration of supported content type.